### ISSUE TITLE
Multiple error messages when using Glue immediately after install

### ISSUE THREAD

**Issue by SCO-SCI** [glue-viz/glue#2498]
Immediately after install the latest version of Anaconda on my Windows 10 desktop, I created a Python 3.9 environment and installed Glue into that environment according to the instructions on the website for "Anaconda Python Distribution - Command line Installation" as shown here: 

https://docs.glueviz.org/en/stable/installation/conda.html

There were no errors during any of this work.

I then started up Glue from the command line inside the Conda environment desi_env by typing 'glue'.  Still no errors.  I then imported a .csv file into Glue using the 'import' button at the top of the Glue window.  I then dragged the bar representing the data onto the main Glue screen and selected '3D scatter plot'.  The result of doing this was a series of error messages that are shown in the attached text file.  Also, the cube that outlines the 3D scatter plot remained empty on the screen and no data was shown on the screen.

Operating System: Windows 10 Pro Version 22H2
Python Version: 3.9.19
Anaconda Version: conda 24.5.0
Glue Version: 1.2.2
Installed using command: conda install -c glueviz glueviz=1.2

Attached is a text file showing the error messages as well as a screen print showing what the Glue window looks like.  Also attached is the .csv file I imported into Glue that resulted in the errors.  I reproduced the error on a different session just now so I could save the session.  The session .glu file would not upload unless I changed the extension to .txt.  That is also attached.

[Glue_Error_Messages.txt](https://github.com/user-attachments/files/16070298/Glue_Error_Messages.txt)

[prepared_data_for_glue.csv](https://github.com/user-attachments/files/16070553/prepared_data_for_glue.csv)

[session2.txt](https://github.com/user-attachments/files/16070737/session2.txt)

![Glue_Error_Window](https://github.com/glue-viz/glue/assets/172367531/744caf81-2bcd-4b0a-9d6c-7da0ef80fd84)

**Comment by SCO-SCI** [comment]
I made some progress on this problem.  I updated vispy using the command shown below.  That eliminated all the error messages reported earlier.  This does seem a bit strange because I ran the command 'conda update -c glueviz --all' earlier and that did not fix the problem.  Nevertheless, although all the error messages are now gone, one problem remains.  When I click on a dot in a scatter plot,  information about that data point should appear in the left panel (I am told).  But that does not happen.  When I click or double-click on a dot in a 2D or 3D scatter plot, nothing happens.  I would appreciate any insight on how to fix that problem.

(desi_env) C:\Users\Ed>conda update vispy
Channels:
 - defaults
 - glueviz
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\Ed\anaconda3\envs\desi_env

  added / updated specs:
    - vispy


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    vispy-0.12.1               |   py39h7c5dd0a_0         1.4 MB
    ------------------------------------------------------------
                                           Total:         1.4 MB

The following packages will be UPDATED:

  vispy                 glueviz::vispy-0.9.3-py39h5d4886f_1 --> pkgs/main::vispy-0.12.1-py39h7c5dd0a_0


Proceed ([y]/n)? y


Downloading and Extracting Packages:

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

**Comment by astrofrog** [comment]
>  When I click on a dot in a scatter plot, information about that data point should appear in the left panel (I am told). But that does not happen. When I click or double-click on a dot in a 2D or 3D scatter plot, nothing happens. I would appreciate any insight on how to fix that problem.

I don't think this is meant to happen, so everything may be working as expected - just to check, where did you read about the click on point behavior you mention?

I think we should update the minimum version of vispy based on the issue you were having.

**Comment by astrofrog** [comment]
I've bumped the minimum version of vispy in https://github.com/glue-viz/glue-vispy-viewers/pull/387

**Comment by SCO-SCI** [comment]
Yes, that is correct.  I was given inaccurate information.  Thank you.

Ed
